README.md

```diff
@@ -299,6 +299,30 @@ This application accepts sample H264/Opus frames and prints them out. To run:
 ./samples/kvsWebrtcClientViewer <channelName>
 ```
 
+#### Sample: Generating sample frames
+
+##### H264
```

Comment:
Can we put these in a summary / details element?

Reply:
Will take care of it in the subsequent PRs